### PR TITLE
[DOC] removeAllDateSuffix update

### PR DESCRIPTION
**Pull request by CerenB**
updated new PR with a correct branch.

### CODE REVIEW

**Review by Remi-Gau — APPROVED** [review]
Looks good to me. :wink:

File: src/utils/removeDateSuffix.m [hunk 1]
```diff
@@ -2,27 +2,21 @@
...
+    %   removeDateSuffix(filenames, subjectPath)
     %
-    % removeDateSuffix(filenames, subjectPath)
+    % :param filenames: File names with _date- suffix
+    % :type filenames: string
     %
+    % :param subjectPath: Location/path of the files
+    % :type subjectPath: string
     %
+    % :output: - files are renamed by removing '_date-yyyymmddHHMM' suffix
```
Comment (Remi-Gau) [inline src/utils/removeDateSuffix.m:19]:
```suggestion
    % :output: - files are renamed by removing '_date-*' suffix
```

Actually I was just checking the `cfg` structure and it is possible to change the format of that date to be something else than the default one you have here. So maybe let's put a "joker" in there.

**Review by Remi-Gau — CHANGES_REQUESTED** [review]

File: src/utils/removeAllDateSuffix.m [hunk 2]
```diff
@@ -2,26 +2,25 @@
...
-    %           - :argout2: (type) (dimension)
-    %
-    % removeAllDateSuffix(rawDir, subjName, sesName)
+    % :param subjName: Some of the options can be ``sub-001`` or ``sub-pilot001``.
+    % :type subjName: string
     %
+    % :param sesName: Some of the options can be ``ses-001`` or ``ses-003``.
+    % :type sesName: string
+
+    % :output: - files are renamed by removing '_date-yyyymmddHHMM' suffix
```
Comment (CerenB) [inline src/utils/removeAllDateSuffix.m:22]:
```suggestion
    % :output: - files are renamed by removing '_date-*' suffix
```

File: src/utils/removeAllDateSuffix.m [hunk 3]
```diff
@@ -2,26 +2,25 @@
 
 function removeAllDateSuffix(rawDir, subjName, sesName)
     %
-    % Short description of what the function does goes here.
+    % Function removes the date suffix in the _events and _stim (.tsv and .json)
+    % files in given raw, session and subject folder. And zips the _stim files.
+    % Function will look for files with suffix of ``_date-yyyymmddHHMM``
```
Comment (CerenB) [inline src/utils/removeAllDateSuffix.m:7]:
```suggestion
    % Function will look for files with suffix of ``_date-*``
```

**Review by marcobarilari — APPROVED** [review]

**Pull request merged**